Author: Henry-Nicolas Tourneur <debian@nilux.be>
Forwarded: not-needed
Description: Fix an FTBFS with meson 0.60.0
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/data/meson.build
+++ b/data/meson.build
@@ -63,7 +63,6 @@ configure_file(
 desktop_conf = configuration_data()
 desktop_conf.set('APP_ID', application_id)
 desktop_file = i18n.merge_file(
-  'desktop',
   input: configure_file(
     input: meson.project_name() + '.desktop.in.in',
     output: '@BASENAME@',
@@ -90,7 +89,6 @@ appdata_conf = configuration_data()
 appdata_conf.set('APP_ID', application_id)
 appdata_conf.set('GETTEXT_PACKAGE', gettext_package)
 appdata_file = i18n.merge_file(
-  'appdata',
   input: configure_file(
     input: meson.project_name() + '.appdata.xml.in.in',
     output: '@BASENAME@',
